src/sia/services/data_collections.py

```diff
@@ -0,0 +1,105 @@
+"""Data collection helper service."""
+
+from dataclasses import dataclass
+
+from ..config import Config
+from ..exceptions import FatalFaultError, UsageFaultError
+from ..models.data_collections import ButlerDataCollection
+
+
+@dataclass
+class DataCollectionService:
+    """Data Collection service class."""
+
+    config: Config
+
+    def get_default_collection(
+        self,
+    ) -> ButlerDataCollection:
+        """Return the default Data collection.
+
+
+        Returns
+        -------
+        ButlerDataCollection
+            The default Butler Data collection.
+
+        Raises
+        ------
+        FatalFaultError
+            If no default Data collection is found.
+        """
+        for collection in self.config.butler_data_collections:
+            if collection.default:
+                return collection
+
+        raise FatalFaultError(
+            detail="No default Collection found. Please configure a default."
+        )
```

Comment:
I would move this check to a validator on `Config` so that the application fails at startup if this is not correct. Then you can write this whole thing as something like `next(c for c in self.config.butler_data_collecitons if c.default)`.

Reply:
I've now removed default from the attributes of a Data collection, following a discussion between Tim & Gregory where it seems that the requirement is now a SIAv2 service will work with a single Butler repository.